xonsh/completer.py

```diff
@@ -362,6 +383,7 @@ def _load_bash_complete_files(self):
 
     def attr_complete(self, prefix, ctx):
         """Complete attributes of an object."""
+        _ = self
```

Comment:
Please remove this line, make this a static method, or by make this method into a function at global scope.